### ISSUE TITLE
Would be nice, if you can add a github actions example

### ISSUE THREAD

**Issue by Ismoh** [moteus/luacov-coveralls#29]
I was struggling with github actions, until I found this after days:

https://github.com/moteus/luacov-coveralls/issues/21#issuecomment-574577498

Would be nice to add this in your readme, for all the ppl struggling with this!

Have a look on this:
https://github.com/moteus/luacov-coveralls/issues/21#issuecomment-1027364557

**Comment by alerque** [comment]
Why was this closed? It seems reasonable to add example usage on Actions to the README to match the current Travis example.

**Comment by Ismoh** [comment]
Yea, you're right. I closed it, because there is a reference issue, where you can find additional infos, but while double checking this issue, I recognised the initial problem I had, isn't mentioned.

So:
Would be nice to add the github actions setup in readme.md

**Comment by Ismoh** [comment]
@moteus I could fork this repo and create a pull request, if you like.

**Comment by Ismoh** [comment]
Is this repo dying?

**Comment by alerque** [comment]
No the repo isn't dying. The maintainer is not very active, but when things need fixing or contributions come in they do *eventually* get dealt with. In the mean time the project works fine in its current form.

If you want a current example of this in use maybe checkout the [`busted` testing workflow for my fluent-lua project](https://github.com/alerque/fluent-lua/blob/master/.github/workflows/busted.yml) and it's [coverage output here](https://coveralls.io/github/alerque/fluent-lua). Note that this workflow successfully aggregates coverage results across all the test runs on different Lua versions.

**Comment by Ismoh** [comment]
I dont get this running on Windows. Do you know an example of github action using windows?

**Comment by alerque** [comment]
No, I don't use Windows at all in real life are rarely in testing. I'm not familiar with the issues running Lua based stuff on it. I suggest if you think something is wrong with the project you open a specific issue related to whatever Windows problem you are seeing. This issue asking for a documentation example is fair enough but understandably low priority for a volunteer maintainer. Given that I've shown a working GitHub Actions example in use, solving this documentation thing won't solve your issue anyway. Leave this here for the docs to be addressed eventually, but post specific details of your Windows problem if necessary.

**Comment by Ismoh** [comment]
Yea, sure thing! Thank you for the answers, have a nice one!
